src/test/java/org/fcrepo/camel/integration/FcrepoContainerPreferIT.java

```diff
@@ -184,20 +184,6 @@ public void configure() {
                         "/rdf:RDF/rdf:Description/ldp:contains", ns)
                     .to("mock:filter");
 
-                from("direct:preferHeadersCheckServerManaged")
-                    .to(fcrepo_uri)
-                    .filter().xpath(
-                        "/rdf:RDF/rdf:Description/fedora:uuid", ns)
-                    .to("mock:filter");
-
-                from("direct:includeServerManaged")
-                    .to(fcrepo_uri + "?preferInclude=ServerManaged")
-                    .filter().xpath(
-                        "/rdf:RDF/rdf:Description/fedora:uuid", ns)
-                    .to("mock:filter")
-                    .to(fcrepo_uri)
-                    .to("mock:container");
-
```

Comment:
There is another uuid reference on (old) line 210 or (new) line 196. Can you change that to reference the ID field instead?